processing/src/main/java/io/druid/query/groupby/epinephelinae/RowBasedGrouperHelper.java

```diff
@@ -78,7 +80,9 @@
     Preconditions.checkArgument(concurrencyHint >= 1 || concurrencyHint == -1, "invalid concurrencyHint");
 
     final GroupByQueryConfig querySpecificConfig = config.withOverrides(query);
+    final DateTime fudgeTimestamp = GroupByStrategyV2.getUniversalTimestamp(query);
```

Comment:
can we add a comment here for future readers to understand why we want to exclude the timestamp?